tests/test_wheel.py

```diff
@@ -6,15 +6,15 @@
 import subprocess
 import sys
 import sysconfig
-import importlib.metadata
 
 import pytest
 import wheel.wheelfile
 
 try:
+    import importlib.metadata  # not available in Python 3.7
```

Comment:
Why not just use `importlib_metadata` in that case?

https://github.com/python/importlib_metadata

Reply:
(a) because I didn't know it existed, and (b) I used `importlib.metadata` to avoid a dependency (could have used https://packaging.pypa.io/en/latest/version.html for version comparison too, but decided not to). I think it's fine as is - and I don't really care about extra Python 3.7 testing given that it's already dropped in SciPy and other similar projects), but happy to change if you prefer to add a new test dependency.